DICOM/src/Microsoft.Health.Dicom.Anonymizer.CommandLineTool.UnitTests/AnonymizerToolUnitTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Dicom;
+using Microsoft.Health.Dicom.Anonymizer.CommandLineTool;
+using Microsoft.Health.Dicom.Anonymizer.Core.Exceptions;
+using Xunit;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.UnitTests
+{
+    public class AnonymizerToolUnitTests
+    {
+        public static IEnumerable<object[]> GetInvalidCommandLine()
+        {
+            yield return new object[] { "-i DicomFiles/I341.dcm" };
+            yield return new object[] { "-o DicomFiles/I341.dcm" };
+            yield return new object[] { "-I DicomFiles/I341.dcm" };
+            yield return new object[] { "-O DicomFiles/I341.dcm" };
+            yield return new object[] { "-i DicomFiles/I341.dcm -o DicomFiles/I341.dcm" };
+            yield return new object[] { "-I DicomFiles -O DicomFiles" };
+        }
+
+        [Fact]
+        public async Task GivenOneDicomFile_WhenAnonymize_ResultWillBeReturnedAsync()
+        {
+            var commands = "-i DicomFiles/I341.dcm -o I341.dcm";
+            await AnonymizerCliTool.Main(commands.Split());
+            var dicomFile = await DicomFile.OpenAsync("I341.dcm");
```

Comment:
💡  [Discussion] 

Don’t know if your tests has some time constraint or disk access privilege issue or not, but opening a real file in UTs may bring some problems.